core/src/test/scala/unit/kafka/security/auth/ZkAuthorizationTest.scala

```diff
@@ -55,9 +56,11 @@ class ZkAuthorizationTest extends ZooKeeperTestHarness with Logging{
   @Test
   def testIsZkSecurityEnabled() {
     assertTrue(JaasUtils.isZkSecurityEnabled())
+    Configuration.setConfiguration(null)
```

Comment:
@fpj Before the changes in this PR, this test used to check only the System property. Now JaasUtils.isZkSecurityEnabled() loads the JAAS configuration to check if ZK security is configured. Hence configuration needs to be reset to force reload the next time configuration is loaded for the next test.